src/components/EventList/EventList.tsx

```diff
@@ -73,50 +122,30 @@ const EventList = () => {
     }
   };
 
-  const filterEventsByTitle = (events: Event[], query: string): Event[] => {
-    return events.filter((event) =>
-      event.title.toLowerCase().includes(query.toLowerCase())
-    );
+  const filterEvents = (events: Event[], query: string): Event[] => {
```

Comment:
@arunimaChintu Where is the functionality to search by title?

Reply:
It was actually searching by tags,,... Okay! am updating it to searching by title